.storybook/main.ts

```diff
@@ -0,0 +1,59 @@
+import path from 'path';
+import remarkGfm from 'remark-gfm';
+
+const toPath = (_path: string) => path.join(process.cwd(), _path);
+
+module.exports = {
+  staticDirs: [toPath('.storybook/public')],
+  stories: [
+    '../packages/circuit-ui/**/*.@(mdx|stories.@(js|ts|tsx))',
+    '../docs/**/*.@(mdx|stories.@(js|ts|tsx))',
+  ],
+  addons: [
+    {
+      name: '@storybook/preset-typescript',
+      options: {
+        transpileManager: true,
+      },
+    },
+    {
+      name: '@storybook/addon-docs',
+      options: {
+        mdxPluginOptions: {
+          mdxCompileOptions: {
+            remarkPlugins: [remarkGfm],
+          },
+        },
+      },
+    },
+
+    '@storybook/addon-storysource',
+    '@storybook/addon-controls',
+    '@storybook/addon-actions',
+    '@storybook/addon-a11y',
+    '@storybook/addon-links',
+    '@storybook/addon-viewport',
+    '@storybook/addon-interactions',
+  ],
+  core: {
+    disableTelemetry: true,
+  },
+  framework: {
+    name: '@storybook/react-webpack5',
+    options: {},
+  },
+  babel: async (options) => ({
+    ...options,
+    presets: [
+      ...options.presets,
+      // HACK: Storybook includes `@babel/preset-react` by default, which
+      // overrides the custom preset configuration in `babel.config.json`.
+      // This override overrides the override.
```

Comment:
😂

Reply:
Maybe I misunderstood the migration guide (only skimmed through), but could we use the new [`jsxOptions`](https://github.com/storybookjs/storybook/blob/next/MIGRATION.md#dropped-addon-docs-manual-babel-configuration) instead? (but I do love this comment)

Reply:
I was very proud when I wrote this 😄

Reply:
I completely missed that option; thanks for pointing it out! I'll give it a shot.

Reply:
I tried adding...

```ts
  jsxOptions: {
    runtime: 'automatic',
    importSource: '@emotion/react',
  },
```

...in `main.ts`, with no success. The migration guide mentions this under the `addon-docs`, so wonder if this is specific to docs mode.

Reply:
Yeah I was also wondering this. Let's keep the wonderful comment 👌